app/views/announcements/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<header>
+  HEADER
+</header>
+
+<body>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3 sidebar">
+        <h1>Announcements</h1>
+
+          <% @announcements.each do |a| %>
+            <div class="post">
+              <div class="title"><%= a.subject %></div>
+              <div class="date">Posted on <%= a.created_at.localtime.strftime("%d %B %C%y, %I:%M %p") %></div>
```

Comment:
In fact you used `created_at` here, not the `date` lol. So don't need the `date` column.

Also, abstract the `strftime` method in the `Announcements` helper. (`app/helpers/announcements_helper.rb`, you can define a method there, and you can use that method here)